### ISSUE TITLE
False-positive "A second item with name ... imported" for scoped vs textual macros

### ISSUE THREAD

**Issue by dima74** [intellij-rust/intellij-rust#6774]
## Environment

* **IntelliJ Rust plugin version:** 0.3.141.3634-203-nightly
* **Rust toolchain version:** 1.51.0-nightly (1d0d76f8d 2021-01-24) x86_64-unknown-linux-gnu
* **IDE name and version:** CLion 2020.3.1 (CL-203.6682.181)
* **Operating system:** Linux 5.10.10-arch1-1
* **Macro expansion engine:** new
* **Name resolution engine:** old

## Problem description
![image](https://user-images.githubusercontent.com/6505554/106492376-e8637100-64c8-11eb-9ed0-d0abd201be66.png)

## Steps to reproduce
`main.rs`:
```rust
fn main() {}

use empty_rust::foo;
macro_rules! foo { () => {} }
```
`lib.rs`:
```rust
#[macro_export]
macro_rules! foo { () => {} }
```

---

Probably we should filter out textual scoped macros here:

https://github.com/intellij-rust/intellij-rust/blob/ef5bdff0f188357ebe4eb078a69147d91a3cc501/src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt#L1297
